frontend/src/app/search/search.component.html

```diff
@@ -210,6 +211,7 @@ <h4 class="mat-subheader">By Topic</h4>
           (mapClick)="closeInfoWindow()"
           (zoomChange)="updateZoom($event)"
         >
+          <agm-marker-cluster [imagePath]="'assets/map/m'" [maxZoom]="8">
```

Comment:
We removed the clustering before because there was some concern about confusion based on one of their user-feedback sessions. Is there a compelling reason to add it back in? And if so, perhaps we should consider some sort of instructions on hover or something like that?